packages/headless/src/features/query-suggest/query-suggest-slice.ts

```diff
@@ -76,7 +79,6 @@ export const querySuggestReducer = createReducer(
           return;
         }
 
-        querySuggest.q = '';
```

Comment:
Potentially only semi-breaking change, we need an option that clears the suggestions but not the query, currently we don't have this, but it can be added and this one kept for potential users.

Reply:
If the source of truth for the query in the query set, wouldn't `clearQuerySuggest` effectively just clear the suggestions?